[PATCH] fix build with --disable-crypto

options.c is missing the definition for struct context when built with
--disable-crypto, as it then doesn't get pulled in through push.h,
leading to build errors like:

options.c: In function ‘parse_http_proxy_fallback’:
options.c:1474: error: dereferencing pointer to incomplete type
options.c:1477: error: dereferencing pointer to incomplete type
options.c:1478: error: dereferencing pointer to incomplete type

Fix it by including forward.h

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 options.c |    1 +
 1 file changed, 1 insertion(+)

Index: openvpn-2.1.3/options.c
===================================================================
--- openvpn-2.1.3.orig/options.c
+++ openvpn-2.1.3/options.c
@@ -29,6 +29,7 @@
 
 #include "syshead.h"
 
+#include "forward.h"
 #include "buffer.h"
 #include "error.h"
 #include "common.h"
